src/renderer/api/endpoints/limit-range.api.ts

```diff
@@ -0,0 +1,52 @@
+import { KubeObject } from "../kube-object";
+import { KubeApi } from "../kube-api";
+import { autobind } from "../../utils";
+
+export enum LimitType {
+  CONTAINER = "Container",
+  POD = "Pod",
+  PVC = "PersistentVolumeClaim",
+}
+
+export enum Resource {
+  MEMORY = "memory",
+  CPU = "cpu",
+  STORAGE = "storage",
+  EPHEMERAL_STORAGE = "ephemeral-storage",
+}
+
+export interface ILimitRangeItem {
+  type: string;
+  max?: { [key: string]: string };
+  min?: { [key: string]: string };
+  default?: { [key: string]: string };
+  defaultRequest?: { [key: string]: string };
+  maxLimitRequestRatio?: { [key: string]: string };
+}
```

Comment:
Having thee limit names would be useful. Furthermore, interfaces don't need to be prefixed with `I`.

```suggestion
export enum LimitPart {
  MAX = "max",
  MIN = "min",
  DEFAULT = "default",
  DEFAULT_REQUEST = "defaultRequest",
  MAX_LIMIT_RATIO = "maxLimitRequestRatio",
}

type LimitRangeParts = Partial<Record<LimitPart, Record<string, string>>>;

export interface LimitRangeItem extends LimitRangeParts {
  type: string;
}
```